_inc/lib/class.core-rest-api-endpoints.php

```diff
@@ -2020,6 +2029,15 @@ public static function prepare_options_for_response( $module = '' ) {
 				$options = self::split_options( $options, get_option( 'verification_services_codes' ) );
 				break;
 
+			case 'google-analytics':
+				$wga = get_option( 'wga' );
```

Comment:
Is the option name the same as the existing `WP Google Analytics` plugin? If so, should we give it a unique name to avoid conflicts / issues on sites that already use that plugin?

Instead of creating a new option, maybe we could add that to the existing Jetpack Options?